tests/unit/test_file_handle_cache.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tests the file handle cache
+"""
+
+from ga4gh import datamodel
+
+import unittest
+import tempfile
+import os
+
+import ga4gh.frontend as frontend
```

Comment:
Why do we need to import the frontend here? I think it would be better to test this class in isolation.